src/static/styles/App.scss

```diff
@@ -510,6 +511,18 @@ hr {
 	border-color: white;
 }
 
+.signupbutton-size{
+	border-bottom-left-radius: 0rem;
+	border-top-left-radius: 0rem;
+	
+}
+
+.emailbutton1-size{
```

Comment:
can you make the classname a little more descriptive like "mailchimp-email-input-button"